doc/thirdpartypackages/index.rst

```diff
@@ -1,179 +1,147 @@
 .. _thirdparty-index:
 
-*********************
- Third party packages
-*********************
+********************
+Third party packages
+********************
 
 Several external packages that extend or build on Matplotlib functionality
-exist. Below we list a number of these. Note that they are each
-maintained and distributed separately from Matplotlib, and will need
-to be installed individually.
+exist. Below we list a number of these. Note that they are each maintained
```

Comment:
Several external packages that extend or build on Matplotlib functionality are
listed below.  They are maintained and distributed separately from Matplotlib
and thus need to be installed individually.

sounds good?